packages/nextjs/app/builders/[address]/page.tsx

```diff
@@ -76,14 +83,44 @@ export default async function BuilderPage({ params }: { params: { address: strin
           <div className="lg:col-span-1">
             <UserProfileCard user={user} address={userAddress} />
           </div>
-          <div className="lg:col-span-3">
+          <div className="lg:col-span-3 flex flex-col gap-14">
             {bgMemberExists && <UpgradedToBGCard user={user} />}
-            <h2 className="text-2xl font-bold mb-0 text-neutral pb-4">Challenges</h2>
-            {challenges.length > 0 ? (
-              <UserChallengesTable challenges={challenges} />
-            ) : (
-              <div className="bg-base-100 p-8 text-center rounded-lg text-neutral">
-                This builder hasn&apos;t completed any challenges.
+            {/* Challenges */}
+            <div className="w-full">
+              <h2 className="text-2xl font-bold mb-0 text-neutral pb-4">Challenges</h2>
+              {challenges.length > 0 ? (
+                <UserChallengesTable challenges={challenges} />
+              ) : (
+                <div className="bg-base-100 p-8 text-center rounded-lg text-neutral">
+                  This builder hasn&apos;t completed any challenges.
+                </div>
+              )}
+            </div>
+            {/* Builds */}
+            {!HIDE_BUILDS && (
+              <div className="w-full">
+                <div className="flex justify-between items-center">
+                  <h2 className="text-2xl font-bold mb-0 text-neutral pb-4">Builds</h2>
+                  <SubmitNewBuildBtn />
```

Comment:
This button should only show for the profile owner. Right now, it's showing on every profile.

Also consider moving all of this section into a component (since this profile page it's going to get bigger and bigger)